current land use enum with URL safe characters

turbomam opened the pull request.

turbomam committed:
current land use enum with URL safe characters

lschriml commented:
As discussed we need to enumerate these longer lists as option values: 
e.g. conifer
       conifer:pine

Instead of having them listed as 'examples' in LinkML.

sujaypatil96 commented on src/mixs/schema/mixs.yaml:
```diff
@@ -25,6 +25,99 @@ subsets:
+      cities:
+      conifers:
+        examples:
+          - value: cypress
```
TIL the practice of adding examples on PV's! ✅

turbomam replied:
Will switch all examples to PVs of their own and assert `is_a` between them (hemlock) and their super-classes (conifer)

sujaypatil96 commented on src/mixs/schema/mixs.yaml:
```diff
@@ -25,6 +25,99 @@ subsets:
 enums:
+  CurLandUseEnum:
+    permissible_values:
+      badlands:
```
Can we add meanings to some of these PV's?

For example, is this an appropriate meaning for the badlands PV? http://purl.obolibrary.org/obo/ENVO_00000127

sujaypatil96 commented on src/mixs/schema/mixs.yaml:
```diff
@@ -25,6 +25,99 @@ subsets:
+  CurLandUseEnum:
+    permissible_values:
+      badlands:
+      cities:
```
Similarly, meaning for cities – http://purl.obolibrary.org/obo/ENVO_00000856

sujaypatil96 reviewed (commented):
Awesome job turning the `string_serialization` on `cur_land_use` to a enumeration. I added a few comments on if/how we can possibly further enrich the `CurLandUseEnum` (by adding meanings)?

turbomam committed:
no PV examples only is_as

turbomam committed:
fix: add aliases for renamed enum values in CurLandUseEnum

mines/quarries → mines-quarries and roads/railroads → roads-railroads
were renamed for URL safety but the old forms need aliases so existing
data remains valid.

turbomam commented on src/mixs/schema/mixs.yaml:
```diff
@@ -110,6 +110,126 @@ enums:
+        description: evergreen forest receiving greater than 406 cm annual rainfall
+      rangeland:
+      roads-railroads:
+      rock:
```
Fixed — added `aliases` for the old `/` forms (`mines/quarries`, `roads/railroads`) so existing data remains valid.

turbomam committed:
Merge remote-tracking branch 'origin/main' into 969-cur_land_use-was-an-enum-before-linkml

turbomam committed:
Merge main into 969-cur_land_use-was-an-enum-before-linkml
